src/style.css

```
@@ -1 +1,38 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

it is not a good practice to use `*` selector for styles reseting. For reseting default styles you can use `body` tag
```suggestion
body {
  margin: 0;
}
```